### PR TITLE
Replace MetaData, RunInfo and Trace class with central data-model-lib

### PR DESCRIPTION
**Pull request by KochTobi**
* replace MetaData by life.qbic.datamodel.workflows.MetaData
* replace RunInfo by life.qbic.datamodel.workflows.RunInfo
* replace Trace by life.qbic.datamodel.workflows.Trace

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,211 +1,219 @@
...
-            <phase>process-test-sources</phase>
-            <goals>
-              <goal>testCompile</goal>
-            </goals>
-            <configuration>
-              <compilerId>groovy-eclipse-compiler</compilerId>
-            </configuration>
-          </execution>
-        </executions>
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sven1103) [inline pom.xml:1]: What happened with this file? Do you use tabs or spaces for intendation?

File: src/main/groovy/life/qbic/flowstore/database/MariaDBStorage.groovy [hunk 2]
```diff
@@ -164,7 +165,7 @@ class MariaDBStorage implements Workflows, AutoCloseable{
 
     private Integer storeRunInfo(RunInfo runInfo) {
         def primaryKey
-        if( isRunInfoStored(runInfo) ) {
+        if (isRunInfoStored(runInfo)) {
```
Comment (sven1103) [inline src/main/groovy/life/qbic/flowstore/database/MariaDBStorage.groovy:168]: the bracket belongs syntactically to the `if` condition, therefore i try to do spaces like this

**Review by sven1103 — APPROVED** [review]
Nice Tobi 👍

File: pom.xml (hunk 1 above)
Comment (KochTobi) [reply pom.xml:1]: 4 spaces. Same as in the groovy classes.

**Pull request merged**